browser/src/core/Socket.js

```diff
@@ -1409,20 +1402,21 @@ app.definitions.Socket = L.Class.extend({
 		}
 		else if (this._reconnecting) {
 			// we are reconnecting ...
-			this._reconnecting = false;
-			this._map._docLayer._resetClientVisArea();
-			this._map._docLayer._requestNewTiles();
+			this._map._docLayer._refreshTilesInBackground();
 			this._map.fire('statusindicator', {statusType: 'reconnected'});
-			this._map._isNotebookbarLoadedOnCore = false;
 			var uiMode = this._map.uiManager.getCurrentMode();
-			this._map.fire('changeuimode', {mode: uiMode, force: !window.migrating});
+			this._map.fire('changeuimode', {mode: uiMode, force: false});
```

Comment:
we call `this.map.sendUnoCommand('.uno:ToolbarMode?Mode:string=notebookbar_online.ui');`  only onAdd currently in notebookbar. it has to be done here to be sure notebookbar's widgets work